wolfcrypt/test/test.c

```diff
@@ -1980,6 +1980,15 @@ WOLFSSL_TEST_SUBROUTINE int md2_test(void)
         if (XMEMCMP(hash, test_md2[i].output, MD2_DIGEST_SIZE) != 0)
             return -1500 - i;
     }
+    
+    for (i = 0; i < times; ++i){
+        wc_Md2Hash((byte*)test_md2[i].input, (word32)test_md2[i].inLen, hash);
```

Comment:
We should check return codes of all function calls for functions that return non-void.